### PR TITLE
Translation to Swahili

### PR DESCRIPTION
**Pull request by the1Riddle**
merging the Swahili translated version to the main repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,41 @@
-# Complete list of all GitHub Profile badges and Achievements
+# Orodha kamili ya beji zote za Mafanikio katika Wasifu wa GitHub
 
-#### Displaying achievements on your profile is completely optional; by default, they can be seen by anyone viewing your public profile.
+#### Kuonyesha mafanikio yako kwenye wasifu wako ni hiari kabisa; kwa chaguo msingi, zinaweza kuonekana na mtu yeyote anayetazama wasifu wako wa umma.
```
Comment (notquitelikeme) [inline README.md:3]: *yanaweza

File: README.md [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
 <br>
 
-## Badge Tiers
+## Vyeo vya Beji
 
-Some Achievements not only have the base version, but also tiers.
+Baadhi ya mafanikio hayana toleo la msingi pekee, bali pia vyeo.
 
-| Achievement | Default | Bronze | Silver | Gold |
+| Mafanikio | Chaguomsingi | Shaba | Silver | Dhahabu |
```
Comment (notquitelikeme) [inline README.md:29]: Silver translation in Swahili - Fedha

**Review by notquitelikeme — COMMENTED** [review]
Minor grammar flaws but translation very much on point.

File: README.md (hunk 1 above)
Comment (the1Riddle) [reply README.md:3]: will work on to fix this, thanks for the review

**Comment by notquitelikeme** [comment]
👍

**Comment by the1Riddle** [comment]
hello @notquitelikeme ,
can you give this a review #2

Thanks

**Comment by notquitelikeme** [comment]
Minor issues fixed

**Pull request merged**